ql/ABQLRowSaveCore.js

```diff
@@ -0,0 +1,20 @@
+/*
+/*
+ * ABQLRowSaveCore
+ *
+ * An ABQLRowSave can store the current Data set into the Process Task it is
+ * in, so that this data can be made available to other Process Tasks.
+ *
+ */
+
+const ABQLSetSave = require("../../platform/ql/ABQLSetSave.js");
+
+class ABQLRowSaveCore extends ABQLSetSave {
+}
+
+ABQLRowSaveCore.key = "row_save";
+ABQLRowSaveCore.label = "save";
+ABQLRowSaveCore.NextQLOps = [];
```

Comment:
I know in my setting I left off having ```Save``` operations ending the chain.  But we will want to keep the chain going here.  We want to support a scenerio like:
```
STEP                              RESULT
Find User Where uuid=="pong"      SET, [ {Pong} ]
SetFirst                          ROW, {Pong}
RowPluck "Emails"                 SET, [ {Email1->Pong}, {Email2-Pong}, ]
SetSave  "All Emails"             SET, [ {Email1}, {Email2}, ]
SetFirst                          Row, {Email1}
RowPluck "address"                VALUE, "Email1.address"
ValueSave "pong's email"          VALUE, "Email1.address"
ValueModify XXX                   VALUE, "Email1.address.modified"
ValueSave "pong's email modified" VALUE, "Email1.address.modified"
```

So we will need to chain existing commands off of the ```Save``` operations.

```Save``` operations don't change the type of values, so 
- ```SetSave``` return ```Set``` operations
- ```RowSave``` return ```Row``` operations
- ```ValueSave``` return ```Value``` operations